mono/mini/aot-compiler.c

```diff
@@ -2763,6 +2763,40 @@ arch_emit_imt_trampoline (MonoAotCompile *acfg, int offset, int *tramp_size)
 
 	code = buf;
 
+#if defined(TARGET_OSX)
+	/* Load the mscorlib got address */
```

Comment:
Can all these changes be simplified by defining a tmp_reg or something which is r11 on apple and r12 elsewhere ?